components/director/internal/domain/runtime/service.go

```diff
@@ -147,6 +164,13 @@ func (s *service) Create(ctx context.Context, in model.RuntimeInput) (string, er
 		in.Labels[model.ScenariosKey] = model.ScenariosDefaultValue
 	}
 
+	scenarios, err := s.synchroniseScenariosWithAssignments(ctx, in.Labels)
```

Comment:
As Adam pointed out in his docs, we probably shouldn't keep DEFAULT scenario on the list